avalancheup-aws/src/main.rs

```diff
@@ -255,6 +255,13 @@ async fn main() -> io::Result<()> {
                     .get_one::<String>("VM_BINARY_FILE")
                     .unwrap_or(&String::new())
                     .to_string(),
+                subnet_validate_period_in_days: *sub_matches
+                    .get_one::<u64>("SUBNET_VALIDATE_PERIOD_IN_DAYS")
+                    .unwrap_or(&14),
+                subnet_config_file: sub_matches
+                    .get_one::<String>("SUBNET_CONFIG_FILE")
+                    .unwrap_or(&String::new())
+                    .to_string(),
```

Comment:
While it doesn't really matter for stuff like this, I would argue that it's just as easy to apply allocation best practices even where performance doesn't really matter for code consistency. 

```rust
                subnet_validate_period_in_days: sub_matches
                    .get_one::<u64>("SUBNET_VALIDATE_PERIOD_IN_DAYS")
                    .copied()
                    .unwrap_or(14),
                subnet_config_file: sub_matches
                    .get_one::<String>("SUBNET_CONFIG_FILE")
                    .cloned()
                    .unwrap_or_else(|| String::new()),
```

And one should always use `unwrap_or_else` if there's a function call even though the compiler would optimize things away in this case.